app/controllers/import_issues_controller.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+class ImportIssuesController < ApplicationController
+  before_action :set_programme
+  before_action :set_vaccination_record, only: %i[show update]
+  before_action :set_form, only: %i[show update]
+
+  def index
+    @import_issues = @programme.import_issues
+
+    render layout: "full"
```

Comment:
Here we can set `layout "full"` on the controller since every action uses this layout.